source/JsMaterialX/.gitignore

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Commenting on the new file `shaderball.glb` above, was this included in the JavaScript changelist by accident?

Reply:
This is the gltf version of the shaderball to use for browser testing. It's read in by three.js.